apps/web/src/pages/SearchRequests/SearchPage/components/SearchPools.tsx

```diff
@@ -58,28 +85,64 @@ const SearchPools = ({
           },
         )}
       </p>
+      <p data-h2-margin="base(x.5, 0, x1, 0)">
+        {" "}
+        <ExternalLink
+          mode="inline"
+          type="button"
+          color="secondary"
+          href={paths.pool(pool.id || "")}
+          newTab
+        >
+          {intl.formatMessage({
+            defaultMessage: " View the job poster for this recruitment process",
+            id: "UvdxsU",
+            description:
+              "Link message that shows the job poster for the recruitment process.",
+          })}
+        </ExternalLink>
+      </p>
+      <p data-h2-margin="base(x1, 0, 0, 0)">
+        {intl.formatMessage({
+          defaultMessage:
+            "These essentials skills were assessed during the process:",
+          id: "NFoOcU",
+          description:
+            "Text showing the essentials skills assessed during the process",
+        })}
+      </p>
+      <p>{skillsArray}</p>
       <p data-h2-margin="base(x1, 0, 0, 0)">
         {intl.formatMessage(
           {
-            defaultMessage: "Pool Owner: {name}",
-            id: "o+a0IN",
-            description: "Text showing the owner of the HR pool.",
+            defaultMessage: "Process run by {team} at {departments}",
+            id: "6yDdnk",
+            description: "Text showing the team and department of the HR pool.",
           },
           {
-            name: pool?.owner
-              ? getFullNameHtml(
-                  pool?.owner.firstName,
-                  pool?.owner.lastName,
-                  intl,
+            team: pool?.team
+              ? getLocalizedName(pool.team.displayName, intl)
+              : intl.formatMessage({
+                  defaultMessage: "Digital Community Management Team",
+                  id: "gfO156",
+                  description: "Text showing the team of the HR pool.",
+                }),
+            departments: departmentsArray
+              ? departmentsArray.join(
+                  intl.formatMessage({
+                    defaultMessage: " and ",
```

Comment:
Shouldn't this be comma separated if it is a list of departments? It is also missing a `description` value to provide context for translation.

Reply:
Jerbo asked for it like that in the issue . I was unsure about the description here but the request is sent already. I can add some here anyways.

Reply:
the spacing seems to have some issue around the "and"

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/ccccf10d-bfe6-4011-9d81-aad75288c25d)

Reply:
aah.. should be fixed  in https://github.com/GCTC-NTGC/gc-digital-talent/pull/6586/commits/898e976d53eafd716a3ffbd4fd88f13586b346ff. Thanks!

Reply:
Yeah, seems to have fixed it. My guess is that `intl.formatMessage({})` removes all whitespacing.